rero_ils/modules/documents/templates/rero_ils/_request_button.html

```diff
@@ -0,0 +1,71 @@
+{# -*- coding: utf-8 -*-
+
+  RERO ILS
+  Copyright (C) 2020 RERO
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+#}
+<!-- action button : Should be at the end to be render above and be clickabke -->
+{%- if item|item_and_patron_in_same_organisation %}
+  {%- set can_request, reasons = item|can_request %}
+  {%- set locations = item|item_library_pickup_locations %}
+  {%- if can_request and locations %}
+    <dd class="{% if class %}{{ class }} {% endif %}col-lg-12 col-sm-12 mt-2">
+      <a
+        href="#"
+        type="button"
+        class="btn btn-primary btn-mini"
+        data-toggle="dropdown"
+        aria-haspopup="true"
+        aria-expanded="false"
+        id="{{ item.barcode }}-dropdownMenu"
+      >
+        {{ _('Request') }}
+        <i class="fa fa-caret-down fa-fw"></i>
+      </a>
+      <!-- TODO: Style the dropdown header -->
```

Comment:
I could be mistaken, but i think that this TODO could be removed. I'm not sure it is still relevant.

Reply:
done